tests/API_FB.py

```diff
@@ -34,12 +34,16 @@ def extract_jobs_title(soup):
 
 
 def create_soup(req):
-    print(req.status_code)
     soup = BeautifulSoup(req.content, 'html.parser')
     # print(soup.prettify())
     return soup
 
 
+def check_status_code(req):
+    print(req.status_code)
```

Comment:
What is the purpose of this function as opposed to just using req.status_code directly?

Reply:
for the PyTests, I chagnes the URL testing to valid URL instead of static URLs..

Reply:
The only thing that this function does is print the status_code (which you don't need to do) and return it.
In the tests function for example:
`assert API_FB.check_status_code(API_FB.create_req()) == 200, "The tests for URLs were successful"`

It should be 
`assert API_FB.create_req().status_code == 200, "The tests for URLs were successful"`

Reply:
Just saw your review, I'll fix it